plugin.rb

```diff
@@ -48,11 +48,24 @@
 =end
   reviewable_api_enabled = defined?(Reviewable)
 
+
+  # We need to migrate existing settings automatically
+  # TODO: remove when 2.4 becomes the new stable
+  group_list_changed = (SiteSetting.group_list_changed rescue false)
```

Comment:
You are following the same pattern in multiple places, just define a constant in plugin.rb instead. then you don't need to recheck this or handle another exception. 

Also... instead of this, SiteSetting.respond_to? :group_list_changed ?